pkg/flag/misconf_flags.go

```diff
@@ -55,6 +55,12 @@ var (
 		Default:    []string{},
 		Usage:      "specify paths to override the Terraform tfvars files",
 	}
+	CfParamsFlag = Flag{
+		Name:       "cf-params",
+		ConfigName: "misconfiguration.cloudformation.vars",
```

Comment:
nit: `cloudformation.vars` is a bit different from the CLI flag. Is it intended? `cloudformation.params` seems more aligned.